model/riscv_fdext_regs.sail

```diff
@@ -270,6 +270,15 @@ function wF_bits(i: bits(5), data: flenbits) -> unit = {
 
 overload F = {rF_bits, wF_bits, rF, wF}
 
+val rF_or_X_H : bits(5) -> bits(16) effect {escape, rreg}
+function rF_or_X_H(i) = {
+  assert(sizeof(flen) >= 32);
```

Comment:
As far as this function's concerned it only needs to be 16 (the assert is so that the function always type-checks even if F registers are too small for nan_unbox of them to be able to yield a 16-bit value). Arguably that's not necessary for this or for the _S version due to the architecture not having a way to have floating-point with FLEN < 32 (Zfh requires F) but it's there for completeness in case it's ever needed, and definitely is needed in the _D case for F-without-D systems.